src/components/Editor/EditorMenu.js

```diff
@@ -67,9 +65,8 @@ async function EditorMenu(
 
   const menuOptions = [];
 
-  if (isMapped) {
-    menuOptions.push(jumpLabel);
-  }
+  // TODO: Find a new way to only add this for mapped sources?
+  menuOptions.push(jumpLabel);
```

Comment:
yeah - this will be tough, the way we're doing it now. But that is okay. This will just be a no-op in the context menu.